engine/schema/src/main/resources/META-INF/db/schema-41910to41920.sql

```diff
@@ -21,3 +21,6 @@
 
 -- Add last_id to the volumes table
 CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.volumes', 'last_id', 'bigint(20) unsigned DEFAULT NULL');
+
+ALTER TABLE `cloud`.`counter` DROP KEY `uc_counter__provider__source__value`;
+CALL `cloud`.`IDEMPOTENT_ADD_UNIQUE_KEY`('cloud.counter', 'uc_counter__provider__source__value_removed', '(provider, source, value, removed)');
```

Comment:
how will we deal with this in upgrade paths that bypass this one?
I mean can we add an if-exists construct and copy it to future upgrades?

Reply:
If I understand correctly, what Daan meant is the upgrade from a version which does not have this change , for example 4.20.0.0 ?

Reply:
Ah.. got it.. I completely forgot :facepalm: